src/pages/Build.css

```
@@ -0,0 +1,598 @@
+  font-size: 11px;
+  color: #555;
+  margin-left: 8px;
+}
+
+.b-prompt-preview {
+  padding: 20px 24px;
+  font-family: ui-monospace, monospace;
+  font-size: 13px;
+  line-height: 1.8;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)